Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/library/SurveilXTest.java

```diff
@@ -0,0 +1,96 @@
+package org.mage.test.cards.abilities.oneshot.library;
+
+import mage.cards.Card;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mage.test.player.TestPlayer;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author grimreap124
+ */
+public class SurveilXTest extends CardTestPlayerBase {
```

Comment:
rather than a new class, can you just put the new tests into the existing `SurveilTest`? then less duplicated code